modules/nf-core/merquryfk/merquryfk/tests/main.nf.test

```diff
@@ -22,63 +22,35 @@ nextflow_process {
                 ]
                 """
             }
-            }
-            run("FASTK_FASTK", alias: "FASTK_MAT") {
+        }
+
+        run("FASTK_FASTK", alias: "FASTK_MAT") {
             script "../../../fastk/fastk"
             process {
                 """
                 input[0] = [
-                    [ id:'test', single_end:false ], 
+                    [ id:'test_mat', single_end:false ],
                     file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
                 ]
                 """
             }
-            }
+        }
```

Comment:
does MerquryFK require hapmaker ktabs as input in nf-test?

Reply:
It will run with any ktab - though it wouldn't be a coherent analysis without hapmaker ktabs. But for a test I think this is fine